File: 03.Shared Preferences/MySharedPreference/app/src/main/java/com/dicoding/picodiploma/mysharedpreference/UserModel.java
package com.dicoding.picodiploma.mysharedpreference;

public class UserModel {

    String name;
    String email;
    int age;
    String phoneNumber;
    boolean isLove;

    public void setName(String name) {
        this.name = name;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public void setAge(int age) {
        this.age = age;
    }

    public void setPhoneNumber(String phoneNumber) {
        this.phoneNumber = phoneNumber;
    }

    public void setLove(boolean love) {
        isLove = love;
    }

    public String getName() {
        return name;
    }

    public String getEmail() {
        return email;
    }

    public int getAge() {
        return age;
    }

    public String getPhoneNumber() {
        return phoneNumber;
    }

    public boolean isLove() {
        return isLove;
    }
}

File: Shared Preferences/MySharedPreference/kotlin/app/src/main/java/com/dicoding/picodiploma/mysharedpreference/UserModel.kt
package com.dicoding.picodiploma.mysharedpreference

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

@Parcelize
class UserModel (
        var name: String? = null,
        var email: String? = null,
        var age: Int = 0,
        var phoneNumber: String? = null,
        var isLove: Boolean = false
) : Parcelable